gax-java/gax/src/main/java/com/google/api/gax/rpc/Watchdog.java

```diff
@@ -95,11 +100,22 @@ private void start() {
             this, scheduleInterval.toMillis(), scheduleInterval.toMillis(), TimeUnit.MILLISECONDS);
   }
 
+  /**
+   * Backport of {@link #watch(ResponseObserver, java.time.Duration, java.time.Duration)} using
+   * {@link org.threeten.bp.Duration}
+   */
+  @ObsoleteApi("Use watch(ResponseObserver, java.time.Duration, java.time.Duration) instead")
+  public <ResponseT> ResponseObserver<ResponseT> watch(
+      ResponseObserver<ResponseT> innerObserver,
+      @Nonnull org.threeten.bp.Duration waitTimeout,
+      @Nonnull org.threeten.bp.Duration idleTimeout) {
+    return watch(innerObserver, toJavaTimeDuration(waitTimeout), toJavaTimeDuration(idleTimeout));
+  }
   /** Wraps the target observer with timing constraints. */
   public <ResponseT> ResponseObserver<ResponseT> watch(
       ResponseObserver<ResponseT> innerObserver,
-      @Nonnull Duration waitTimeout,
-      @Nonnull Duration idleTimeout) {
+      @Nonnull java.time.Duration waitTimeout,
+      @Nonnull java.time.Duration idleTimeout) {
```

Comment:
public method -- I don't think we can change types

Reply:
We do have an overload here, so we preserve the original `threeten` version.

https://github.com/googleapis/sdk-platform-java/blob/3c9edbb78102263f5089dfbb8fc1ed0b674af6e7/gax-java/gax/src/main/java/com/google/api/gax/rpc/Watchdog.java#L103-L113

Reply:
I will rename the `java.time` version of `watch` and `create` to `watchDuration` and `createDuration` respectively.

Reply:
Done